src/test/java/com/xpandit/plugins/xrayjenkins/Utils/ProxyUtilTest.java

```diff
@@ -0,0 +1,130 @@
+package com.xpandit.plugins.xrayjenkins.Utils;
+
+import com.xpandit.xray.service.impl.delegates.HttpRequestProvider;
+import hudson.ProxyConfiguration;
+import jenkins.model.Jenkins;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Jenkins.class, ProxyConfiguration.class})
+@PowerMockIgnore({"javax.net.ssl.*", "javax.management.*"})
+public class ProxyUtilTest {
+
+    private static final String PROXY_DOMAIN = "proxydomain.com";
+    private static final int PROXY_PORT = 8080;
+    private static final String JIRA_DOMAIN = "jiradomain.com";
+    private static final int JIRA_PORT = 8080;
+    private static final String JIRA_HTTP_URL = "http://" + JIRA_DOMAIN;
+    private static final String JIRA_HTTP_URL_WITH_PORT = "http://" + JIRA_DOMAIN + ":" + JIRA_PORT;
```

Comment:
Please add a "legacy check" to the unit tests:
Before this update, the "No Proxy Host" was in the following format: *www.my-jira-server.com/*
Please check this old format still works and is still being ignored